db/migrate/20220429080952_create_canonical_clothing_items.rb

```diff
@@ -4,7 +4,7 @@ class CreateCanonicalClothingItems < ActiveRecord::Migration[6.1]
   def change
     create_table :canonical_clothing_items do |t|
       t.string :name, null: false, unique: true
-      t.decimal :unit_weight, precision: 5, scale: 1, default: 1.0
```

Comment:
It was better to remove a default unit weight since unit weights vary enough there's no real reason to assume that would be the weight of any particular item.